src/migrations/hotbar-store/5.0.0-alpha.0.ts

```diff
@@ -2,18 +2,22 @@
 import { Hotbar } from "../../common/hotbar-store";
 import { clusterStore } from "../../common/cluster-store";
 import { migration } from "../migration-wrapper";
+import { v4 as uuid } from "uuid";
 
 export default migration({
   version: "5.0.0-alpha.0",
   run(store) {
     const hotbars: Hotbar[] = [];
 
     clusterStore.enabledClustersList.forEach((cluster: any) => {
-      const name = cluster.workspace || "default";
+      const name = cluster.workspace;
```

Comment:
We should remove the workspace from the cluster after this migration.

Reply:
Let's keep it here for a while.... just because someone might still want to go back to 4.x series after testing 5.x.

Reply:
okay sounds good. Maybe once we release 5.0?